### ISSUE TITLE
Infinite loop when serializing feature flags

### ISSUE THREAD

**Issue by cmanallen** [getsentry/sentry-python#3862]
### How do you use Sentry?

Sentry Saas (sentry.io)

### Version

2.18

### Steps to Reproduce

```python
from sentry_sdk._lru_cache import LRUCache

cache = LRUCache(3)
cache.set(0, 0)

copied = copy(cache)
copied.set(2, 2)
cache.get_all()  # spins forever
```

### Expected Result

The loop breaks and flags are returned.

### Actual Result

Infinite loop.

**Comment by cmanallen** [comment]
This is fixed by: https://github.com/getsentry/sentry-python/pull/3861

We missed a deepcopy on the LRU.  Added a deepcopy and am now using an explicitly terminating to prevent this error.

**Comment by antonpirker** [comment]
This has been released with version `2.19.2`: https://github.com/getsentry/sentry-python/blob/master/CHANGELOG.md#2192
